device/bgp/bgpsrteadvanced.yaml

```diff
@@ -0,0 +1,25 @@
+components:
+  schemas:
+    BgpSrTe.Advanced:
+      description: >-
+        Configuration for advanced BGP route range settings.
+      type: object
+      properties:
+        origin:
+          description: >-
+            The origin value. 
+            A null value means the origin feature is not enabled.
```

Comment:
A null values implies this feature is not enabled.

Reply:
done